packages/cloud_functions/cloud_functions/android/src/main/java/io/flutter/plugins/firebase/functions/FlutterFirebaseFunctionsPlugin.java

```diff
@@ -156,11 +166,19 @@ private Map<String, Object> getExceptionDetails(@Nullable Exception exception) {
 
   @Override
   public Task<Map<String, Object>> getPluginConstantsForFirebaseApp(FirebaseApp firebaseApp) {
-    return Tasks.call(() -> null);
+    TaskCompletionSource<Map<String, Object>> taskCompletionSource = new TaskCompletionSource<>();
```

Comment:
Since `getPluginConstantsForFirebaseApp` returns a `Task<Map<String, Object>> ` I think it's correct to keep `TaskCompletionSource<Map<String, Object>>`

Reply:
Yeah, sorry, that can stay. I just saw that it always returns `null` and it didn't make sense to me why we would return a Map. But I guess it is an overridden method so no choice.